pkg/costattribution/active_tracker.go

```diff
@@ -138,15 +198,19 @@ func (at *ActiveSeriesTracker) Decrement(lbls labels.Labels) {
 	at.observedMtx.RLock()
 	as, ok := at.observed[string(buf.Bytes())]
 	if ok {
-		nv := as.Dec()
+		nv := as.asCount.Dec()
+		if nativeHistogramBucketNum >= 0 && as.nhCount.Load() > 0 {
+			as.nhCount.Dec()
+			as.nhBucketNum.Sub(int64(nativeHistogramBucketNum))
+		}
 		if nv > 0 {
 			at.observedMtx.RUnlock()
 			return
 		}
 		at.observedMtx.RUnlock()
 		at.observedMtx.Lock()
 		as, ok := at.observed[string(buf.Bytes())]
```

Comment:
I think this `as` variable everywhere is now `counters` (or maybe just `c`? but not `as`)

Reply:
that commit did not change this line, still called `as`

Reply:
addressed in 47deaae69ffe17c051c46b4ecd9b9ddf67ab4c6c